src/app/destinyTrackerApi/perkRater.js

```diff
@@ -0,0 +1,109 @@
+import _ from 'underscore';
+
+/**
+ * Rate perks on an item (based off of its attached user reviews).
+ *
+ * @class PerkRater
+ */
+class PerkRater {
+  /**
+   * Rate the perks on an item based off of its attached user reviews.
+   *
+   * @param {any} item
+   * @memberof PerkRater
+   */
+  ratePerks(item) {
+    if ((!item.writtenReviews) ||
+        (!item.writtenReviews.length)) {
+      return;
+    }
+
+    const maxColumn = this._getMaxColumn(item);
+
+    for (let i = 1; i < maxColumn; i++) {
+      const perkNodesInColumn = this._getPerkNodesInColumn(item, i);
+
+      const ratingsAndReviews = _.map(perkNodesInColumn, (perkNode) => this._getPerkRatingsAndReviewCount(perkNode, item.writtenReviews));
+
+      const maxReview = this._getMaxReview(ratingsAndReviews);
+
+      this._markNodeAsBest(maxReview);
+    }
+  }
+
+  _markNodeAsBest(maxReview) {
+    if (!maxReview) {
+      return;
+    }
+
+    maxReview.perkNode.bestRated = true;
+  }
+
+  _getMaxReview(ratingsAndReviews) {
+    const orderedRatingsAndReviews = _.sortBy(ratingsAndReviews, (ratingAndReview) => { return ratingAndReview.ratingCount < 2 ? 0 : ratingAndReview.averageReview; }).reverse();
+
+    if ((orderedRatingsAndReviews.length > 0) &&
+        (orderedRatingsAndReviews[0].ratingCount > 1)) {
+      return orderedRatingsAndReviews[0];
+    }
+
+    return null;
+  }
+
+  _getMaxColumn(item) {
+    return _.max(item.talentGrid.nodes, (node) => { return node.column; }).column;
```

Comment:
Same deal here - this could be `_.max(item.talentGrid.nodes, (node) => node.column)`